packages/did-core-test-server/suites/did-production/did-jsonld-production.js

```diff
@@ -20,16 +20,18 @@ const generateJsonldProductionTests = (
   });
 
   it('6.3.1 JSON-LD Production - The serialized value of @context ' +
-    'MUST be the JSON String https://www.w3.org/ns/did/v1, or a JSON ' +
+    'MUST be the JSON String https://www.w3.org/ns/did/v1 (DID v1.0) or '+
+    'https://www.w3.org/ns/did/v1.1 (DID v1.1), or a JSON ' +
     'Array where the first item is the JSON String ' +
-    'https://www.w3.org/ns/did/v1 and the subsequent items are serialized ' +
+    'https://www.w3.org/ns/did/v1 or https://www.w3.org/ns/did/v1.1, '+
+    'and the subsequent items are serialized ' +
     'according to the JSON representation production rules.', async () => {
       const context = didDocument['@context'];
 
       if(typeof context === 'string') {
-        expect(context).toBe('https://www.w3.org/ns/did/v1');
+        expect(context).toMatch(/^https\:\/\/www.w3.org\/ns\/did\/v1|https\:\/\/www.w3.org\/ns\/did\/v1\.1$/);
       } else if(Array.isArray(context)) {
-        expect(context[0]).toBe('https://www.w3.org/ns/did/v1');
+        expect(context[0]).toMatch(/^https\:\/\/www.w3.org\/ns\/did\/v1|https\:\/\/www.w3.org\/ns\/did\/v1\.1$/);
```

Comment:
If `toBe()` was the wrong assertion because it was doing something like checking a type, then maybe there is a `toEqual()` instead of `toMatch()` to simplify.

Reply:
The problem is that jest doesn't support matching multiple strings, it's either a single value, you pull in a separate library that adds a  `.oneOf()` match, or you use a regex. I went with a regex because pulling in the extra library would require an update to package-lock.json, which would trigger a whole bunch of upgrades necessary across the entire test infrastructure.

The current infrastructure is pretty terrible -- it's over-engineered and in need of a significant re-write. Once we have did-resolution working, we should probably use that instead of the current test suite. That would require significant effort to re-write, so we're having to make suboptimal choices in order to reduce this cycle of testing.